EmailPiece.dart

```diff
@@ -0,0 +1,51 @@
+import 'dart:core';
+import 'package:enough_mail/enough_mail.dart';
+import 'package:intl/intl.dart';
+import 'package:summer2022/models/Digest.dart';
+
+class EmailPiece {
+  String EmailPieceID; // Unique ID of Image
+  String EmailID; // Unique ID of Email - made from Email contents hashed
+  DateTime EmailPieceTimestamp;
+  String EmailPieceSender;

```

Comment:
I believe these can all just be set as "public" to expose them directly rather than having the separate "return" getter methods.